### PR TITLE
Revert "ENT-3658, ENT-3660: Add timeouts and hospital flow handling to CryptoService

### PR DESCRIPTION
**Pull request by fowlerrr**
This reverts commit 6df142bf

### CODE REVIEW

**Comment by shamsasari** [comment]
You will need to rebase as there's conflicts.

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt [hunk 1]
```diff
@@ -3,16 +3,11 @@ package net.corda.nodeapi.internal.cryptoservice
...
 
 class CryptoServiceFactory {
     companion object {
-        fun makeCryptoService(
-                cryptoServiceName: SupportedCryptoServices,
-                legalName: CordaX500Name,
-                signingCertificateStore: FileBasedCertificateStoreSupplier? = null,
-                timeout: Duration? = null
-        ): CryptoService {
+        fun makeCryptoService(cryptoServiceName: SupportedCryptoServices, legalName: CordaX500Name,  signingCertificateStore: FileBasedCertificateStoreSupplier? = null, cryptoServiceConf: Path? = null): CryptoService {
```
Comment (shamsasari) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt]: Let's keep the parameters on separate lines, both to keep this line short and add to better manage conflicts in the params.

Comment (shamsasari) [inline node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt]: `cryptoServiceConf` is not used so you can remove it. I wouldn't worry too much about diff with ENT as ENT already as extra parameters.

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt [hunk 2]
```diff
@@ -175,11 +176,7 @@ abstract class AbstractNode<S>(val configuration: NodeConfiguration,
     val transactionStorage = makeTransactionStorage(configuration.transactionCacheSizeBytes).tokenize()
     val networkMapClient: NetworkMapClient? = configuration.networkServices?.let { NetworkMapClient(it.networkMapURL, versionInfo) }
     val attachments = NodeAttachmentService(metricRegistry, cacheFactory, database, configuration.devMode).tokenize()
-    val cryptoService = CryptoServiceFactory.makeCryptoService(
-            SupportedCryptoServices.BC_SIMPLE,
-            configuration.myLegalName,
-            configuration.signingCertificateStore
-    ).closeOnStop()
+    val cryptoService = CryptoServiceFactory.makeCryptoService(SupportedCryptoServices.BC_SIMPLE, configuration.myLegalName, configuration.signingCertificateStore)
```
Comment (shamsasari) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Let's keep the parameters on separate lines, as it's with ENT

Comment (fowlerrr) [reply node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Done

File: node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt (hunk 1 above)
Comment (fowlerrr) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt]: Done

Comment (fowlerrr) [reply node-api/src/main/kotlin/net/corda/nodeapi/internal/cryptoservice/CryptoServiceFactory.kt]: Done

**Pull request merged**